src/services/milestones/milestones.hooks.js

```diff
@@ -32,7 +33,7 @@ const restrict = () => (context) => {
       if (!reviewers.includes(user.address)) throw new errors.Forbidden('Only the reviewer accept or cancel a milestone');
 
       // whitelist of what the reviewer can update
-      const approvedKeys = ['txHash', 'status', 'mined'];
+      const approvedKeys = ['txHash', 'status', 'mined', 'prevStatus'];
```

Comment:
see below comment. If we want to keep this, do we need to store it? It looks like we're only using it to trigger notifications.

Maybe it's cleaner to just store it, thoughts?